metals/src/main/scala/scala/meta/internal/metals/BloopInstall.scala

```diff
@@ -215,7 +215,12 @@ object BloopInstall {
     val plugins =
       if (sbt.version.startsWith("0.13")) pluginsDirectory("0.13")
       else pluginsDirectory("1.0")
-    Files.createDirectories(plugins.toNIO)
+    val isSymlink = Files.isSymbolicLink(plugins.toNIO) &&
+      Files.isDirectory(Files.readSymbolicLink(plugins.toNIO))
+    // createDirectories checks whether the directory exists, but it doesn't follow symlinks
+    if (!isSymlink) {
+      Files.createDirectories(plugins.toNIO)
```

Comment:
Can we create an extension method `AbsolutePath.createDirectories()` in `MetalsEnrichments` that performs this check? It would make it easier to reuse in the future.